plugins/permission-node/src/integration/createPermissionIntegrationRouter.ts

```diff
@@ -274,17 +333,34 @@ export function createPermissionIntegrationRouter<
 >(
   options:
     | { permissions: Array<Permission> }
-    | CreatePermissionIntegrationRouterResourceOptions<
-        TResourceType,
-        TResource
-      >,
+    | CreatePermissionIntegrationRouterResourceOptions<TResourceType, TResource>
+    | Array<CreatePermissionIntegrationRouterResourceOptions<string, any>>,
```

Comment:
There's an argument to be made that `{ resources: CreatePermissionIntegrationRouterResourceOptions<...> }` might be a bit more future proof, rather than allowing an array of the existing options. With this setup it gets trickier to supply options that are not specific to each resource.